.github/workflows/ci.yml

```diff
@@ -8,6 +8,9 @@ on:
     tags:
       - "v[0-9]+.[0-9]+.[0-9]+*"
   pull_request:
+    branches:
```

Comment:
This would prevent the pipeline from running whenever we use feature branches on the main repo.

Reply:
we use feature branches?

Reply:
not often, but it is an option for us if we need it

Reply:
for cases when we have big features that we need to add piece by piece and want to keep main releasable